File: app/src/main/java/com/djavid/bitcoinrate/model/cryptocompare/ConversionType.java
package com.djavid.bitcoinrate.model.cryptocompare;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class ConversionType {

    @SerializedName("type")
    @Expose
    public String type;

    @SerializedName("conversionSymbol")
    @Expose
    public String conversionSymbol;


    public String getType() {
        return type;
    }
    public void setType(String type) {
        this.type = type;
    }

    public String getConversionSymbol() {
        return conversionSymbol;
    }
    public void setConversionSymbol(String conversionSymbol) {
        this.conversionSymbol = conversionSymbol;
    }
}

File: app/src/main/kotlin/com/djavid/bitcoinrate/model/cryptocompare/ConversionType.kt
package com.djavid.bitcoinrate.model.cryptocompare

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


class ConversionType {

    @SerializedName("type")
    @Expose
    var type: String

    @SerializedName("conversionSymbol")
    @Expose
    var conversionSymbol: String
}
